### PR TITLE
fbugfix:  fix sncloud credentials prompt when using sambastudio

### PR DESCRIPTION
**Pull request by snova-jorgep**

### CODE REVIEW

**Review by snova-kwasia — APPROVED** [review]
LGTM @snova-jorgep

File: search_assistant/streamlit/app.py [hunk 1]
```diff
@@ -167,7 +168,8 @@ def main() -> None:
         st.markdown('Get your SerpApi key [here]( https://serpapi.com)')
 
         if not are_credentials_set(additional_env_vars):
-            api_key, additional_vars = env_input_fields(additional_env_vars)
+            api_key: str = ''
+            api_key, additional_vars = env_input_fields(additional_env_vars, mode=api_key)
```
Comment (snova-rodrigom) [inline search_assistant/streamlit/app.py:172]: is this line ok? I think `mode` would an actual mode value? just double checking

**Review by snova-rodrigom — COMMENTED** [review]
Besides the comment left, LGTM

File: search_assistant/streamlit/app.py (hunk 1 above)
Comment (snova-kwasia) [reply search_assistant/streamlit/app.py:172]: Yes good point @snova-rodrigom 

@snova-jorgep - env_input_fields(additional_env_vars: List[str] = [], mode: str = 'SambaNova Cloud').

Can you take a look? Will hold off until you've checked this.

Comment (snova-jorgep) [reply search_assistant/streamlit/app.py]: right,  fixed

**Pull request merged**